src/renderer/api-decorator.js

```diff
@@ -357,7 +357,17 @@ limitations under the License.
         //---------------------------------------------------------------
         let winOpts = getWindowOptionsSync();
 
-        showOnReady(glbl, winOpts);
+        // Prevent iframes from attempting to do windowing actions, these will always be handled
+        // by the main window frame.
+        if (window.frameElement) {
+
+            // If you are an iframe, you still need to send out the api ready event so frameConnect
+            // can correctly assign connection priority.
+            electron.remote.getCurrentWebContents(renderFrameId).emit('openfin-api-ready', renderFrameId);
```

Comment:
You can move this call out of `showOnReady` and just do it here once

Reply:
👍 updating and retesting